gfx/common/x11_common.c

```diff
@@ -147,6 +153,75 @@ void x11_set_window_attr(Display *dpy, Window win)
 
 void x11_suspend_screensaver(Window wnd, bool enable)
 {
+    x11_suspend_screensaver_xdg_screensaver(wnd, enable);
+#ifdef HAVE_DBUS
+    x11_suspend_screensaver_dbus(enable);
+#endif
+}
+
+#ifdef HAVE_DBUS
+void x11_suspend_screensaver_dbus(bool enable)
+{
+    const char *app = "RetroArch";
+    const char *reason = "Playing a game";
+    DBusMessage *msg, *reply;
+
+    if (!enable)
```

Comment:
> That should uninhibit, not get ignored.

Fair enough, as I said in the comments I was not too sure how the menu option work for enabling / disabling screensaver and if that function gets call with `enabled == false`. I thought I tested it and changing the setting in the menu did not call this function again, but I may be wrong.

Also, I took that straight from the `xdg-screensaver` method, so presumably it suffers from the same issue.